Task 1/Task1.1/Task1.1.3/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task1._1._3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int number;
+            string str = "";
+
+            Console.WriteLine("Enter a number:");
+            number = Convert.ToInt32(Console.ReadLine());
+
+            for(int i = 1; i <= number; i++, str = "")
+            {
+                for (int j = 0; j < number - i; str += ' ', j++) ;
```

Comment:
```suggestion
            for(int i = 1; i <= number; i++)
            {
            str = ""
                for (int j = 0; j < number - i; str += ' ', j++) ;
```
Если я правильно понял, то ты очищаешь переменную в начале каждого цикла, то так будет читабельнее.) Не экономь строки.